docs/source/index.mdx

```diff
@@ -1,138 +1,47 @@
+## What is Apollo Federation?
 
-```mermaid
-graph LR;
-  clients(Clients);
-  subgraph "Supergraph";
-  gateway([Router]);
-  serviceA[Users<br/>subgraph];
-  serviceB[Products<br/>subgraph];
-  gateway --- serviceA & serviceB;
-  end;
-  clients -.- gateway;
-  class clients secondary;
-```
-
-With federation, you can responsibly share ownership of your supergraph across any number of teams and services.  And even if you currently only have _one_ GraphQL API, Apollo Federation is essential for scaling that API as you grow your features, user base, and organization.
-
-[Apollo GraphOS](/graphos/) provides a managed mode for Apollo Federation, which helps you modify and grow your supergraph without any downtime.
-
-<div>
-  <ButtonLink
-    colorScheme="indigo"
-    href="./quickstart/setup/"
-    mr="4"
-  >
-    Get started with Federation
-  </ButtonLink>
-  <Button
-    as="a"
-    href="#how-it-works"
-  >
-    Learn more
-  </Button>
-</div>
-
-## How it works
-
-In a federated architecture, your individual GraphQL APIs are called **subgraphs**, and they're composed into a **supergraph**. By querying your supergraph's router, clients can fetch data from all of your subgraphs with a single request:
+**Apollo Federation** lets you combine multiple GraphQL APIs into a unified graph with a single entry point called the **router**.
+Clients make requests to the router, which intelligently distributes them across your APIs and returns a unified response.
+This architecture enables clients to interact with multiple services with a single request.
 
 ```mermaid
 graph LR;
   clients(Clients);
-  subgraph "Supergraph";
-  gateway([Router]);
-  serviceA[Users<br/>subgraph];
-  serviceB[Products<br/>subgraph];
-  gateway --- serviceA & serviceB;
-  end;
-  clients -.- gateway;
+  router([Router]);
+  serviceA[GraphQL API <br/>A];
+  serviceB[GraphQL API <br/>B];
+  serviceC[GraphQL API <br/>C];
+  router --- serviceA & serviceB & serviceC;
+  clients -.- router;
   class clients secondary;
 ```
 
-The **router** serves as the public access point for your supergraph. It receives incoming GraphQL operations and intelligently routes them across your subgraphs. To clients, this looks exactly the same as querying any other GraphQL server—no client-side configuration is required.
-
-### Combining subgraph schemas
-
-Like any other GraphQL API, each subgraph has its own schema:
-
-<CodeColumns>
-
-```graphql title="Users subgraph"
-type User {
-  id: ID!
-  name: String!
-}
-```
-
-```graphql title="Products subgraph"
-type Product {
-  upc: String!
-  inStock: Boolean!
-}
-```
+For performance and security reasons, clients should only query the router, and only the router should query the constituent APIs.
+For clients, the request and response cycle of querying the router looks the same as querying any GraphQL server.
+No client-side configuration is required.
```

Comment:
Glad you chimed in on this. I went back and forth on this, moving it from the intro to the "Next steps/terminology" section. I'm actually thinking it might fit well under a new Benefits H3 called "Preserve client simplicity" or something like that. LMK if you had other thoughts about where this belongs.